zalando.org_routegroups.yaml

```diff
@@ -34,14 +33,19 @@ spec:
       openAPIV3Schema:
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation
-              of an object. Servers should convert recognized schemas to the latest
-              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: |-
```

Comment:
The change is due to https://github.com/kubernetes-sigs/controller-tools/pull/870